stdlib/public/Darwin/Accelerate/vImage_CGImageFormat.swift

```diff
@@ -0,0 +1,72 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Accelerate
+
+//===----------------------------------------------------------------------===//
+//
+//  vImage_CGImageFormat
+//
+//===----------------------------------------------------------------------===//
+
+@available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
+extension vImage_CGImageFormat {
+    
+    /// Initializes an image format from a Core Graphics image.
+    public init?(cgImage: CGImage) {
+        guard
+            let colorSpace = cgImage.colorSpace else {
+                return nil
+        }
+        
+        self = vImage_CGImageFormat(
+            bitsPerComponent: UInt32(cgImage.bitsPerComponent),
+            bitsPerPixel: UInt32(cgImage.bitsPerPixel),
+            colorSpace: Unmanaged.passRetained(colorSpace),
+            bitmapInfo: cgImage.bitmapInfo,
+            version: 0,
+            decode: nil,
+            renderingIntent: cgImage.renderingIntent)
+    }
+    
+    /// Initializes an image format.
+    public init(bitsPerComponent: UInt32,
```

Comment:
These should be `Int` instead of `UInt32`.

Reply:
Ideally, yes. But the verbose original version of this init uses `UInt32` (see [here](https://developer.apple.com/documentation/accelerate/vimage_cgimageformat/1546793-init)). I kept the same type for my lightweight init for consistency.

Reply:
I don't think "consistency of being a pain in the butt to use" is a very useful guiding principle for API design. =)

Reply:
Good point. Changed to `Int`.